plc4j/drivers/firmata/pom.xml

```diff
@@ -122,12 +122,6 @@
       <version>0.12.0-SNAPSHOT</version>
     </dependency>
 
-    <dependency>
-      <groupId>org.apache.plc4x</groupId>
-      <artifactId>plc4j-transport-serial</artifactId>
```

Comment:
same here, why is the dependency disappearing

Reply:
Because we no longer directly reference serial as we rely on the defaults ...